packages/api/src/beacon/routes/lodestar.ts

```diff
@@ -406,7 +410,7 @@ export function getDefinitions(_config: ChainForkConfig): RouteDefinitions<Endpo
       resp: JsonOnlyResponseCodec,
     },
     getHistoricalSummaries: {
-      url: "/eth/v1/lodestar/historical_summaries/{state_id}",
+      url: "/eth/v1/lodestar/{state_id}/historical_summaries",
```

Comment:
would prefer this to better align with beacon-api

```suggestion
      url: "/eth/v1/lodestar/states/{state_id}/historical_summaries",
```